suit/tests/models.py

```diff
@@ -30,34 +30,47 @@ def __unicode__(self):
         return self.name
 
 
+@admin.register(Book)
 class BookAdmin(admin.ModelAdmin):
-    list_filter = ('id', 'name',)
-    list_display = ('id', 'name',)
+    list_filter = (
+        "id",
+        "name",
+    )
+    list_display = (
+        "id",
+        "name",
+    )
 
     def suit_row_attributes(self, obj, request):
-        return {'class': 'suit_row_attr_class-%s' % obj.name,
-                'data': obj.pk,
-                'data-request': request}
+        return {
+            "class": "suit_row_attr_class-%s" % obj.name,
+            "data": obj.pk,
+            "data-request": request,
+        }
 
     def suit_cell_attributes(self, obj, column):
-        return {'class': 'suit_cell_attr_class-%s-%s' % (column, obj.name),
-                'data': obj.pk}
+        return {
+            "class": "suit_cell_attr_class-%s-%s" % (column, obj.name),
+            "data": obj.pk,
+        }
 
 
+@admin.register(Album)
 class AlbumAdmin(admin.ModelAdmin):
     def suit_row_attributes(self, obj):
         """No request defined to test backward-compatibility"""
-        return {'class': 'suit_row_album_attr_class-%s' % obj.name,
-                'data-album': obj.pk}
+        return {
+            "class": "suit_row_album_attr_class-%s" % obj.name,
+            "data-album": obj.pk,
+        }
 
 
 class User(models.Model):
     """
     Class to test menu marking as active if two apps have model with same name
     """
+
     name = models.CharField(max_length=64)
 
 
-admin.site.register(Book, BookAdmin)
-admin.site.register(Album, AlbumAdmin)
 admin.site.register(User)
```

Comment:
It could, but I'm not sure if worthy though

Reply:
Apparently in the `User` case is not needed because I'm not declaring a `UserAdmin`. This is according to the [docs](https://docs.djangoproject.com/en/5.0/ref/contrib/admin/#modeladmin-objects) in the section _"Do you need a ModelAdmin object at all?"_